Arriba/Arriba/Model/Column/ColumnFactory.cs

```diff
@@ -8,12 +8,86 @@
+            ColumnCreators["int16"] = ColumnCreators["short"];
+
+            ColumnCreators["int"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<int>(details, columnComponents, initialCapacity); };
+            ColumnCreators["int32"] = ColumnCreators["int"];
+
+            ColumnCreators["long"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<long>(details, columnComponents, initialCapacity); };
+            ColumnCreators["int64"] = ColumnCreators["long"];
+
+            ColumnCreators["float"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<float>(details, columnComponents, initialCapacity); };
+            ColumnCreators["single"] = ColumnCreators["float"];
+
+            ColumnCreators["double"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<double>(details, columnComponents, initialCapacity); };
+
+            ColumnCreators["ushort"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<ushort>(details, columnComponents, initialCapacity); };
+            ColumnCreators["uint16"] = ColumnCreators["ushort"];
+
+            ColumnCreators["uint"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<uint>(details, columnComponents, initialCapacity); };
+            ColumnCreators["uint32"] = ColumnCreators["uint"];
+
+            ColumnCreators["ulong"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<ulong>(details, columnComponents, initialCapacity); };
+            ColumnCreators["uint64"] = ColumnCreators["ulong"];
+
+            ColumnCreators["datetime"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<DateTime>(details, columnComponents, initialCapacity); };
+
+            ColumnCreators["guid"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<Guid>(details, columnComponents, initialCapacity); };
+
+            ColumnCreators["timespan"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return Build<TimeSpan>(details, columnComponents, initialCapacity); };
+
+            ColumnCreators["string"] = (details, columnComponents, initialCapacity) => { AdjustColumnComponents(ref columnComponents); return BuildByteBlock(details, columnComponents); };
+            ColumnCreators["json"] = ColumnCreators["string"];
+            ColumnCreators["html"] = ColumnCreators["string"];
```

Comment:
Html uses a different string splitter than String/Json - it looks like that logic didn't get replicated.